src/solver.rs

```diff
@@ -135,13 +137,13 @@ pub fn resolve<P: Package, V: Version>(
                 version: v.clone(),
                 source: err,
             })? {
-            None => {
+            Unavailable => {
```

Comment:
Maybe keeping the `Dependencies::` prefix would be clearer

Reply:
Agree, will change.